src/core/contrib.js

```diff
@@ -52,14 +55,15 @@ export async function run(conf) {
   if (!ghCommenters && !ghContributors) {
     return;
   }
-  const { githubAPI } = conf;
+  const { githubAPI, githubAPIBase } = conf;
```

Comment:
If `githubAPIBase` is always the origin of `conf.githubAPI`, then I don't think we should add it to `conf`. Just do:

```JS
const githubAPIBase = new URL(conf.githubAPI, window.location).origin;
```

And maybe rename `githubAPIBase` to `githubAPIOrigin` so it's more clear.